crates/noirc_errors/src/debug_info.rs

```diff
@@ -1,39 +1,41 @@
+use acvm::acir::circuit::OpcodeLocation;
+use acvm::compiler::AcirTransformationMap;
+
+use serde_with::serde_as;
+use serde_with::DisplayFromStr;
 use std::collections::BTreeMap;
 
 use crate::Location;
 use serde::{Deserialize, Serialize};
 
+#[serde_as]
 #[derive(Default, Debug, Clone, Deserialize, Serialize)]
 pub struct DebugInfo {
     /// Map opcode index of an ACIR circuit into the source code location
-    pub locations: BTreeMap<usize, Vec<Location>>,
+    #[serde_as(as = "BTreeMap<DisplayFromStr, _>")]
```

Comment:
This is needed because although serde in general works for enums, particular serde implementations like Json and Toml, do not support them

Reply:
Exactly! They support them but not for **keys** in json/toml maps/objs, because those need to be strings. That's why a way of solving it is indicating to serde that they key of that map can use to string / from string

Reply:
I'm going to add a comment indicating this